pymongoexplain/explainable_collection.py

```diff
@@ -26,17 +26,27 @@
 
 
 class ExplainableCollection():
-    def __init__(self, collection):
+    def __init__(self, collection, verbosity=None, comment=None):
         self.collection = collection
         self.last_cmd_payload = None
+        self.verbosity = verbosity
+        self.comment = comment
 
     def _explain_command(self, command):
         command_son = command.get_SON()
         explain_command = SON([("explain", command_son)])
-        explain_command["verbosity"] = "queryPlanner"
+        explain_command["verbosity"] = self.verbosity or "queryPlanner"
```

Comment:
I suggest setting the default verbosity in `__init__`.

Reply:
Done.